src/data/faq.json

```diff
@@ -2410,7 +2406,7 @@
                                 "active": false,
                                 "textblock": [
                                     "Yes. As long as you have activated exposure logging in the app and Bluetooth (and Location Services on Android) is on, encounters are logged in the background by the Exposure Notification system integrated in iOS/Android. The app itself doesn't have to be open for this. You can minimize it or use 'swipe-to-quit' to close it.",
-                                    "To be on the safe side, open the app once a day. You'll find more info at <a href='../#howto' target='_blank' >This is how the app works best</a>.",
+                                    "To be on the safe side, open the app once a day.",

```

Comment:
resolve by changing relative link to point to top level of web, not by deleting the sentence.